website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -29,64 +27,120 @@ dbt commands are run against dbt Cloud's infrastructure and benefit from:
+
+It replaces dbt Core, but this change can be avoided by using the native install method and configuring your PATH or by creating a new virtual environment. To switch back to dbt Core, follow the dbt Core installation instructions.
+
+::: 
+
+Before installing the dbt Cloud CLI, make sure you have Python installed and your virtual environment venv or pyenv . If you already have a Python environment configured, you can skip to the [pip installation step](#install-dbt-cloud-cli-in-pip).
+
+### Install a virtual environment
+
+We recommend using virtual environments (venv) to namespace pip modules.
+
+1. Create a new venv:
+   ```shell
+   python3 -m venv dbt-env
+    ```
+
+2. Activate the virtual environment each time you create a shell window or session:
+  ```shell   
+  source dbt-env/bin/activate         # activate the environment for Mac and Linux OR
+  dbt-env\Scripts\activate            # activate the environment for Windows
+  ```
+
+3. Create an alias to activate your dbt environment with every new shell window or session. You can add the following to your shell's configuration file (for example, $HOME/.bashrc, $HOME/.zshrc) while replacing `<PATH_TO_VIRTUAL_ENV_CONFIG>` with the path to your virtual environment configuration:
+
+```shell
+   alias env_dbt='source <PATH_TO_VIRTUAL_ENV_CONFIG>/bin/activate'
+  ```
+
+### Install dbt Cloud CLI in pip
+
+1. (Optional) If you already have dbt Core installed, this installation will override that package. Note your dbt Core version in case you need to reinstall it later:
+
+  ```bash
+  dbt --version
+  ```
+
+2. Run the following command to install the dbt Cloud CLI:
+
+  ```bash
+  pip3 install dbt
+  ```
+
+4. (Optional) To revert back to dbt Core, first uninstall both the dbt Cloud CLI and dbt Core
+5. Reinstall dbt Core using the version from Step 2.
+
+  ```bash
+  pip3 uninstall dbt-core dbt
+  pip3 install dbt-core==VERSION
+  ```
+
+</TabItem>
 	
-<TabItem value="brew" label="macOS">
+<TabItem value="brew" label="macOS (brew)">
+
+Before you begin, make sure you have [Homebrew installed](http://brew.sh/) in your code editor or command line terminal. Refer to the [FAQs](#faqs) if your operating system runs into path conflicts.
 
-Before you begin, make sure you have [Homebrew installed](http://brew.sh/) in your code editor or command line terminal. If your operating system runs into path conflicts, refer to the [FAQs](#faqs).
 
 1. Run the following command to verify that there is no conflict with a dbt Core installation on your system:
    
-```bash
-which dbt
-```
+  ```bash
+  which dbt
+  ```
    - This should return a `dbt not found`. If the dbt help text appears, use `pip uninstall dbt` to deactivate dbt Core from your machine.
   
 2. Install the dbt Cloud CLI with Homebrew: 
 
-```bash
-brew tap dbt-labs/dbt-cli
-brew install dbt-cloud-cli
-```
+  ```bash
+  brew tap dbt-labs/dbt-cli
+  brew install dbt
```

Comment:
per benoit, this command should be same command as update one:

`brew install dbt` or `brew upgrade dbt-cloud-cli` -- is it dbt-cloud-cli or dbt?

Reply:
it is dbt. this one is correct